frontend/src/state/backendApi.ts

```diff
@@ -2112,35 +2112,41 @@ export const rpcnSecretManagerApi = observable({
     const client = appConfig.rpcnSecretsClient;
     if (!client) throw new Error('redpanda connect secret client is not initialized');
 
-    await client.deleteSecret(secret);
+    await client.deleteSecret({ request: secret });
   },
   async create(secret: CreateSecretRequest) {
     const client = appConfig.rpcnSecretsClient;
     if (!client) throw new Error('redpanda connect secret client is not initialized');
 
-    await client.createSecret(secret);
+    await client.createSecret({ request: secret });
   },
   async update(_id: string, updateSecretRequest: UpdateSecretRequest) {
     const client = appConfig.rpcnSecretsClient;
     if (!client) throw new Error('redpanda connect secret client is not initialized');
 
-    await client.updateSecret(updateSecretRequest);
+    await client.updateSecret({ request: updateSecretRequest });
   },
   async checkScope(listSecretScopesRequest: ListSecretScopesRequest) {
     const client = appConfig.rpcnSecretsClient;
     if (!client) throw new Error('redpanda connect secret client is not initialized');
 
-    const scopes = await client.listSecretScopes(listSecretScopesRequest);
-    const isEnable = scopes.scopes.some((scope) => scope === Scope.REDPANDA_CONNECT);
+    const res = await client.listSecretScopes({ request: listSecretScopesRequest });
+
+    if (!res.response) {
+      this.isEnable = false;
+      return;
+    }
+
+    const isEnable = res.response?.scopes.some((scope) => scope === Scope.REDPANDA_CONNECT);
     this.isEnable = isEnable;
     return isEnable;
```

Comment:
I feel like we can improve the `isEnable` flag is set, but it's ok to leave as-is for this PR, there are more refactors we may do later on